ComfortZoneApp/app/src/main/java/com/example/comfortzone/utils/ComfortCalcUtil.java

```diff
@@ -49,17 +53,25 @@ private static int levelWeightCalc(LevelsTracker tracker) {
     }
 
     private static int getSum(LevelsTracker tracker) {
-        ArrayList<ComfortLevelEntry> entryArrayList = (ArrayList<ComfortLevelEntry>) tracker.get(LevelsTracker.KEY_ENTRIESLIST);
-        return entryArrayList.stream().mapToInt(entry -> getTemp(entry)).sum();
+        ArrayList<ComfortLevelEntry> entryArrayList = (ArrayList<ComfortLevelEntry>) tracker.get(KEY_ENTRIESLIST);
+        return entryArrayList.stream().mapToInt(entry -> entry.getTemp()).sum();
```

Comment:
here should you do a null check for entryArrayList and also for entry before accessing entry.getTemp?